src/recharge.iced

```diff
@@ -101,8 +101,10 @@ exports.Runner = class Runner extends Base
 
     fee = @fee_estimator { tx: tx }
 
-    if (@debug)
-      console.log('Expected total fee:', num*@min_amount() + fee)
+    if @debug()
```

Comment:
not sure how accurate this is in general, for example for 1 input/40 outputs we have 1518 +/- 1 total size, but https://blockchain.info/tx/fd973f5956d285a566c1afcbce400ef60159d60202a289842df39c6afba44782 is 1552B